.github/workflows/docker-latest-tag.yml

```
@@ -32,14 +32,13 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Re-tag image
+      - name: Re-tag image and push to GHCR
+        if: ${{ !fromJSON(github.event.inputs.dry_run) }} 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)